# goal
bool turn_on_or_off
---
# result
bool status
---